### PR TITLE
Vagrantfile for single node k8s setup with ADB

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Signed-off-by: Lalatendu Mohanty lmohanty@redhat.com

### CODE REVIEW

File: components/centos/centos-k8s-singlenode-setup/README.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Getting started with vagrantfile
+-----------------------------------------------------------
+
+* This vagrantfile is for setting up a single node kubernetes setup with ADB Vagrant box.
```
Comment (navidshaikh) [inline components/centos/centos-k8s-singlenode-setup/README.rst]: *Vagrantfile

File: components/centos/centos-k8s-singlenode-setup/README.rst [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+* i.e. The master node and node setup in the single machine.
```
Comment (navidshaikh) [inline components/centos/centos-k8s-singlenode-setup/README.rst]: Suggesting to rephrase statement as
i.e. The master and node setup in the single machine.

File: components/centos/centos-k8s-singlenode-setup/README.rst [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+* git clone the adb git repo
+* change in to the directory containing this vagrant file
+* do vagrant up
+
+::
+     
+     git clone https://github.com/projectatomic/adb-atomic-developer-bundle.git
+     cd adb-atomic-developer-bundle/components/centos/centos-with-kubernetes
+     vagrant up
```
Comment (navidshaikh) [inline components/centos/centos-k8s-singlenode-setup/README.rst:18]: I suggest we directly write the steps in commands rather sentences.

Comment (LalatenduMohanty) [inline components/centos/centos-k8s-singlenode-setup/README.rst:18]: @navidshaikh IMO we should describe the steps for better understanding. Sometimes users who are not very experienced might want to know what are the steps do.

File: components/centos/centos-k8s-singlenode-setup/README.rst [hunk 4]
```diff
@@ -0,0 +1,25 @@
+Getting started with vagrantfile
```
Comment (navidshaikh) [inline components/centos/centos-k8s-singlenode-setup/README.rst]: *Vagrantfile

**Comment by navidshaikh** [comment]
LGTM

**Pull request merged**